Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.12.1-alpine3.9 AS builder
+
+COPY . /go/src/github.com/Huawei-PaaS/ci-bot
+
+RUN apk --no-cache update && \
+apk --no-cache upgrade && \
+CGO_ENABLED=1 go build -v -o /usr/local/bin/ci-bot -ldflags="-w -s -extldflags -static" \
```

Comment:
The path `/usr/local/bin/ci-bot` seems does not work for everyone.

Reply:
It will work fine i think
![Screenshot from 2019-04-05 09-50-58](https://user-images.githubusercontent.com/39593583/55604406-46b84f00-578d-11e9-989c-709e3f932ffe.png)

Reply:
oh u mean i must make it generic so that all projects
ex like cni genie, kube volcano ..if they do docker build-image it must build their image (ex kube-volcano) ???? Is that what you need???

Reply:
I mean probably different people may have different go path.